### ISSUE TITLE
The hyperparameter Settings in the script seem to be inconsistent with those in the paper

### ISSUE THREAD

**Issue by leexinhao** [OpenGVLab/VideoMAEv2#21]
Both the learning rate and epoch seem to be different from those described in the paper.

**Comment by congee524** [comment]
`repeated augmentation`  is the cause of the inconsistency.
https://github.com/OpenGVLab/VideoMAEv2/blob/9492db0047a9e30446a4093543a1a39dfe62b459/run_mae_pretraining.py#L208

In the paper, we report the effective epochs (`args.num_sample * args.epochs`) to align with VideoMAE v1 and avoid misunderstanding.
In addition, `lr = base_lr * batch_size / 128`, while the true batch size should be `args.batch_batch * args.num_sample`, so we set `base_lr = 1.5e-4 * 4 = 6e-4`

**Comment by leexinhao** [comment]
Thanks, but why are the epochs of finetuning in scripts much bigger than reported in the paper?
![image](https://github.com/OpenGVLab/VideoMAEv2/assets/48858574/5d5488a7-be20-4036-b38d-ee5c63d1130f)
![image](https://github.com/OpenGVLab/VideoMAEv2/assets/48858574/27b58e15-3b3a-4665-af37-577d340c3386)
![image](https://github.com/OpenGVLab/VideoMAEv2/assets/48858574/cee93274-7315-4e93-b096-ce2c73e2062a)

**Comment by leexinhao** [comment]
OK, thanks for your explanations!
